packages/service-clients/end-to-end-tests/odsp-client/src/test/OdspClientFactory.ts

```diff
@@ -15,43 +15,57 @@ import { OdspTestTokenProvider } from "./OdspTokenFactory";
 /**
  * Interface representing the credentials required for testing odsp-client.
  */
-export interface OdspTestCredentials {
-	clientId: string;
-	clientSecret: string;
+export interface OdspLoginCredentials {
 	username: string;
 	password: string;
 }
 
+export interface OdspCredentials extends OdspLoginCredentials {
```

Comment:
same nit as above, and it would also be nice to have TSDoc comments for this one as well
```suggestion
export interface IOdspCredentials extends IOdspLoginCredentials {
```